src/base/DecodersAndSanitizers/TermFinanceDecoderAndSanitizer.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.21;
+
+import {BaseDecoderAndSanitizer, DecoderCustomTypes} from "src/base/DecodersAndSanitizers/BaseDecoderAndSanitizer.sol";
+
+contract TermFinanceDecoderAndSanitizer is BaseDecoderAndSanitizer {
+    //============================== TERM FINANCE ===============================
+
+    constructor(address _boringVault) BaseDecoderAndSanitizer(_boringVault) {}
+
+    function lockOffers(DecoderCustomTypes.TermAuctionOfferSubmission[] calldata offerSubmissions) pure virtual external returns (bytes memory addressesFound) {
+        for (uint256 i = 0; i < offerSubmissions.length; i++) {
+            addressesFound = abi.encodePacked(
+                addressesFound,
+                offerSubmissions[i].offeror //TODO: Sanitize offeror as vault address?
```

Comment:
I would actually prefer to just report out the offeror address like you are doing. I want to try and move away from Decoders being tied to specific Boring Vaults as its more flexible is any boring vault can use them.

Reply:
what is "report out"?

Reply:
TODO removed